packages/cli/src/swagger/specGenerator3.ts

```diff
@@ -458,6 +467,15 @@ export class SpecGenerator3 extends SpecGenerator {
     return mediaType;
   }
 
+  private buildQueriesParameter(source: Tsoa.Parameter): Swagger.Parameter3[] {
+    if (source.type.dataType === 'refAlias' && source.type.type.dataType === 'nestedObjectLiteral') {
```

Comment:
'refObject' should probably work here. 
We should not only check 1 level deep on the alias, I've written similar code which checks deeep for isVoidType I think

If we can't handle, I'd rather throw than return an array I think?

Reply:
Thinking about this, I think is better to fail if there is a nested object, to be coherent with the `@Query` decorator failing if we pass it a `refObject`. @WoH What do you think?